src/smartpeak/include/SmartPeak/core/SequenceProcessor.h

```diff
@@ -124,6 +135,11 @@ namespace SmartPeak
     ProcessSequenceSegments() = default;
     ProcessSequenceSegments(SequenceHandler& sh) : SequenceProcessor(sh) {}
     void process() const override;
+
+    /* IProcessorDescription */
+    int getID() const { return -1; }
```

Comment:
"override" would be welcome for all these getID() getName() and getDescription()

Reply:
Good point, `override` well describe developer's intention, however its optional